ruleset.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<ruleset>
+    <!-- Contributte Coding Standard -->
+    <rule ref="./vendor/ninjify/coding-standard/contributte.xml">
+        <exclude name="Squiz.Commenting.EmptyCatchComment.Missing"/>
+        <exclude name="Squiz.PHP.Heredoc.NotAllowed"/>
+        <exclude name="Squiz.Strings.DoubleQuoteUsage.ContainsVar"/>
+    </rule>
+</ruleset>
```

Comment:
Pls, let's use ruleset already defined by `gamee/php-code-checker-rules` https://github.com/contributte/datagrid/blob/master/composer.json#L39

Thanks 👍 

(A bit of background: it is used in all packages authored by me)

Reply:
Není lepší Pavle vše mít podle contributte/bare? @f3l1x

Reply:
Sorry but no. I am using those packages along with another 100 private ones that are written this way in 200 projects that are also using mentioned coding standard.

Next time, please ask first. I really appreciate you work and I don't want to waste your time.

And don't worry. I was discussing that with @f3l1x about 3 years ago... it was one of my conditions when migrating under contributte. :)

Reply:
To be clear, those projects will remain using `gamee/php-code-checker-rules` https://packagist.org/packages/gamee/php-code-checker-rules/dependents?order_by=downloads

Maybe we can migrate to it? :) :)

Reply:
ok. I update gamee/php-code-checker-rules  to 2.0.x